src/controller/user-controller.js

```diff
@@ -0,0 +1,51 @@
+const client = require('../db');
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+const jwtTokens = require('../utils/jwt-helpers');
+
+
+
+
+const getAllUserHandler = async (req, res) => {
+    try {
+        client.query(`Select * from users`, (err, result)=>{
+            if(!err){
+                res.send(result.rows);   
+            }
+        });
+        client.end;
+    } catch(error) {
+        res.status(500).json({error: error.message})
+    }
+}
+
+const registerHandler =  async (req, res) => {
+    try {
+        const hashedPassword = await bcrypt.hash(req.body.password, 10);
+        const newUser = await client.query(           
```

Comment:
okay you are creating the user right away, which means people can create with the same email multiple times, are you not suppose to check if the email is taken and also the email should be converted to lower case before being saved